chutney/packaging/local-dev/src/main/resources/application-sso-auth.yml

```diff
@@ -0,0 +1,45 @@
+#SPDX-FileCopyrightText: 2017-2024 Enedis
+#SPDX-License-Identifier: Apache-2.0
+
+spring:
```

Comment:
Maybe have here use parameter value for the provider host + port and another for the redirect url as port 4200 is only valid if you started the ui manually.